daemon/containerd/image_delete.go

```diff
@@ -52,14 +52,17 @@ import (
 // conflict will not be reported.
 //
 // TODO(thaJeztah): image delete should send prometheus counters; see https://github.com/moby/moby/issues/45268
-func (i *ImageService) ImageDelete(ctx context.Context, imageRef string, force, prune bool) (response []imagetypes.DeleteResponse, retErr error) {
+func (i *ImageService) ImageDelete(ctx context.Context, imageRef string, options imagetypes.RemoveOptions) (response []imagetypes.DeleteResponse, retErr error) {
```

Comment:
Looks like the GoDoc needs updating as it's still referencing the `prune` and `force` arguments; while updating, perhaps also remove use of future tense ("will each attempt" -> "are attempted to be deleted quietly")

```go
// cannot be removed unless either there are other repository references to the
// same image or force is true. Following removal of the repository reference,
```

```go
// If prune is true, ancestor images will each attempt to be deleted quietly,
```

Reply:
I adjusted it a bit